tests/vc-generator/index.js

```diff
@@ -0,0 +1,112 @@
+/*!
+ * Copyright 2023 Digital Bazaar, Inc.
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+import * as vc from '@digitalbazaar/vc';
+import {DataIntegrityProof} from '@digitalbazaar/data-integrity';
+import {documentLoader} from './documentLoader.js';
+import {getMultikeys} from './key-gen.js';
+import {getSuite} from './cryptosuites.js';
+import {klona} from 'klona';
+
+/**
+ * Issues test data locally and then returns a Map
+ * with the test data.
+ *
+ * @param {object} options - Options to use.
+ * @param {object} options.credential - An unsigned VC.
+ * @param {string} options.suite - A cryptosuite id.
+ * @param {Array<string>} options.mandatoryPointers - An optional list of
+ *   json pointers.
+ * @param {Array<string>} options.keyTypes - A Set of keyTypes to issue with.
+ *
+ * @returns {Promise<Map<string, object>>} Returns a Map <keyType, vc>.
+ */
+export async function issueTestData({
+  credential,
+  suite,
+  mandatoryPointers,
+  keyTypes = ['P-256']
+}) {
+  const results = new Map();
+  const keys = await getMultikeys({keyTypes});
+  for(const [keyType, {signer, issuer}] of keys) {
+    const _vc = await issueCredential({
+      credential,
+      issuer,
+      signer,
+      suite,
+      mandatoryPointers
+    });
+    results.set(keyType, _vc);
+  }
+  return results;
+}
+
+export async function issueCredential({
+  credential,
+  issuer,
+  signer,
+  suite,
+  mandatoryPointers = []
+}) {
+  const _credential = klona(credential);
+  _credential.issuer = issuer;
+  const cryptosuite = getSuite({suite, mandatoryPointers});
+  return vc.issue({
+    credential: _credential,
+    documentLoader,
+    suite: new DataIntegrityProof({signer, cryptosuite})
+  });
+}
+
+/**
+ * Derives test data locally and then returns a Map
+ * with the test data.
+ *
+ * @param {object} options - Options to use.
+ * @param {object} options.verifiableCredential - A signed VC.
+ * @param {string} options.suite - A cryptosuite id.
+ * @param {Array<string>} options.selectivePointers - An optional list of json
+ *   pointers.
+ * @param {Array<string>} options.keyTypes - A list of key types.
+ *
+ * @returns {Promise<Map<string, object>>} Returns a Map <keyType, vc>.
+ */
+export async function deriveTestData({
+  verifiableCredential,
+  suite,
+  selectivePointers = [],
+  keyTypes = ['P-256']
+}) {
+  const results = new Map();
+  const keys = await getMultikeys({keyTypes});
+  for(const [keyType, {signer}] of keys) {
+    const _vc = await deriveCredential({
```

Comment:
This is a total nit and more a question than a statement. 

I've usually read variables starting with `_` to indicate an unused parameter in a function signature (or to define a pseudo-private method on a class). A local variable in a for block could just be something short, like `vc`?

I'm not sure what the conventions are applied here though, so this is more a question

Reply:
the `_` means its private to the function, but I suppose in this case you have a point as the `_vc` is exported eventually in a Map it might not be entirely private. Maybe in another pass the name can be bikeshed better?